notebooks/Quickstart/Examples/helloworld.wdl

```diff
@@ -0,0 +1,29 @@
+task HWTask {
+
+  String genome_ref_fasta
+  String version = "hello_world_v1.0.0"
+  String runtime_environment
+
+
+  command {
+    echo ${genome_ref_fasta}
+
+    python -u<<CODE
+
+    chars = """Hello World!"""
+    print(chars)
+
+    import sys
+    print(sys.version)
+
+    CODE
+  }
+
+  runtime {
+    docker: (if runtime_environment == "production" then "python:3.6" else "python:2.7") + "-slim"
```

Comment:
Also I think something like `if runtime_environment == "python3" then "python:3.6" else "python:2.7") + "-slim"` would be a more straightforward comparison

Reply:
You could even call it `python_environment`